site/src/main/java/uk/nhs/digital/ps/beans/HippoBeanHelper.java

```diff
@@ -0,0 +1,15 @@
+package uk.nhs.digital.ps.beans;
+
+import org.hippoecm.hst.container.RequestContextProvider;
+import org.hippoecm.hst.content.beans.standard.HippoBean;
+
+/**
+ * Static helper for {@linkplain org.hippoecm.hst.content.beans.standard.HippoBean}
+ */
+public class HippoBeanHelper {
```

Comment:
I'm always a bit wary of introducing catch-all "helper" classes, and I wonder whether this one is worth having for the very small amount of duplication it saves?

Reply:
Leaving for now as discussed.

Reply:
I'm happy with this.